optax/tree_utils/_random.py

```diff
@@ -33,9 +32,9 @@ def _tree_rng_keys_split(
   Returns:
     a tree of rng keys.
   """
-  tree_def = jtu.tree_structure(target_tree)
+  tree_def = jax.tree.structure(target_tree)
```

Comment:
You may keep `jtu` rather than `jax.tree` for the sake of consistency throughout the codebase.